README.md

```diff
@@ -9,8 +9,6 @@ We analyze 40.7 weeks of data, starting right after the MEV grace period ended a
 (2022-11-24 05:35:39Z UTC); see
 https://discord.com/channels/405159462932971535/405163979141545995/1044108182513012796.
 
-
-
 ### Global vs RP sanity check
```

Comment:
Can we avoid "sanity check" here? (E.g. https://source.android.com/docs/setup/contribute/respectful-code for why and suggested alternatives.)